lib/rsolr/client.rb

```diff
@@ -63,6 +65,11 @@ def paginate page, per_page, path, opts = nil
     execute build_paginated_request(page, per_page, path, opts)
   end
   
+  # Sends a query to /select or the custom configured select handler.
+  def select opts={}
```

Comment:
No of course not, that's just an example of what the purpose of the select method would be instead of using ```get '/select'```. Anything that uses RSolr can have the Solr settings in RSolr instead of having the logic to select handlers outside of it.

In any case, if you prefer keeping the logic of selecting which handler to use outside RSolr I can certainly refactor these two pull requests that way.